keyboards/handwired/nicekey/readme.md

```diff
@@ -0,0 +1,18 @@
+# nicekey handwired
+
+![The first NiceKey](https://s3.eu-central-1.amazonaws.com/mindlevel/nicekey.jpg)
+
+
+Custom handwired nicekey, a one key keyboard that writes random compliments.
+
+Keyboard Maintainer:  spydon
```

Comment:
Make sure that these lines end with double spaces, so that it's formatted correctly